README.md

```diff
@@ -1,210 +1,119 @@
 
-### **export_contract_data**
+---
 
-```bash
-> stellar-etl export_contract_data --end-ledger 500000 --output export_contract_data.txt
-```
+# **Command Reference**
 
-Exports historical contract data data from the genesis ledger to the provided end-ledger to an output file. The command reads from the bucket list, which includes the full history of the Stellar ledger. As a result, it should be used in an initial data dump. In order to get contract data information within a specified ledger range, see the export_ledger_entry_changes command.
-
-<br>
+- [Export Commands](#export-commands)
+  - [export_ledgers](#export_ledgers)
+  - [export_transactions](#export_transactions)
+  - [export_operations](#export_operations)
+  - [export_effects](#export_effects)
+  - [export_assets](#export_assets)
+  - [export_trades](#export_trades)
+  - [export_diagnostic_events](#export_diagnostic_events)
+  - [export_ledger_entry_changes](#export_ledger_entry_changes)
+- [Utility Commands](#utility-commands)
+  - [get_ledger_range_from_times](#get_ledger_range_from_times)
 
-### **export_contract_code**
+Every command accepts a `-h` parameter, which provides a help screen containing information about the command, its usage, and its flags.
 
-```bash
-> stellar-etl export_contract_code --end-ledger 500000 --output export_contract_code.txt
-```
+Commands have the option to read from testnet with the `--testnet` flag, from futurenet with the `--futurenet` flag, and defaults to reading from mainnet without any flags.
 
-Exports historical contract code data from the genesis ledger to the provided end-ledger to an output file. The command reads from the bucket list, which includes the full history of the Stellar ledger. As a result, it should be used in an initial data dump. In order to get contract code information within a specified ledger range, see the export_ledger_entry_changes command.
+> _*NOTE:*_ Adding both flags will default to testnet. Each stellar-etl command can only run from one network at a time.
 
 <br>
 
-### **export_config_settings**
+---
 
-```bash
-> stellar-etl export_config_settings --end-ledger 500000 --output export_config_settings.txt
-```
+## **Export Commands**
 
-Exports historical config settings data from the genesis ledger to the provided end-ledger to an output file. The command reads from the bucket list, which includes the full history of the Stellar ledger. As a result, it should be used in an initial data dump. In order to get config settings data information within a specified ledger range, see the export_ledger_entry_changes command.
+These commands export information using the Ledger Exporter output files within a specified datastore (currently [datastore](https://github.com/stellar/go/tree/master/support/datastore) only supports GCS). This allows users to provide a start and end ledger range. The commands in this category export a list of everything that occurred within the provided range. All of the ranges are inclusive.
```

Comment:
may be worthwhile to link Ledger Exporter to its README if someone wants to find out more. Or just move the note that follows below up in the readme

Reply:
Might also be worthwhile to add a note that your datastore must contain the expected XDR files to work, requiring the operator to run a Ledger Exporter to hydrate that data store.